config/300-awsperformanceinsights.yaml

```diff
@@ -98,12 +117,14 @@ spec:
                         - name
                         - key
                     oneOf:
-                    - required: ['value']
-                    - required: ['valueFromSecret']
+                    - required: [value]
+                    - required: [valueFromSecret]
               sink:
+                description: The destination of events generated from Amazon Performance Insights metrics.
```

Comment:
```suggestion
                description: The destination of events generated from the Amazon Performance Insights Source.
```

Reply:
What I wanted to express is that events are generated from metrics, if that makes sense.
An event is generated based on the observed metrics.

How would you express that better?

Reply:
@cab105  we need some proper English guidance xd

Reply:
this is more of a nit. im fine with it as is. it just sounds a bit off to me personally. don't mean it as a blocker we can push past this. ill approve

Reply:
Would that sound better?

"The destination of events generated based on Amazon Performance Insights metrics."

Reply:
lets just remove metrics and go with `The destination of events generated by Amazon Performance Insights` ?

Reply:
You got it

Reply:
Fixed in [0e77c19d..81de3691](https://github.com/triggermesh/aws-event-sources/compare/0e77c19d78c9e9135c6a2d9667a521fddad2cabd..81de3691bd53e353ebe5c3f6f078e2c618106a66)